src/search/containers/AddonPage/index.js

```diff
@@ -13,16 +22,92 @@ class AddonPage extends React.Component {
     slug: PropTypes.string.isRequired,
   }
 
+  dataItems() {
+    const { addon } = this.props;
+    const items = [
+      [addon.type, 'type'],
+      [addon.status, 'status'],
+      [<a href={addon.url} target="_blank">{_('View on site')}</a>, 'url'],
+      [<a href={editUrl(addon.url)} target="_blank">{_('Edit on site')}</a>, 'edit'],
```

Comment:
I filed https://github.com/mozilla/addons-server/issues/2244 and https://github.com/mozilla/addons-frontend/issues/184 to update it.